windows/src/engine/keyman32/keybd_shift.cpp

```diff
@@ -105,33 +105,16 @@ void do_keybd_event(LPINPUT pInputs, int *n, BYTE vk, BYTE scan, DWORD flags, UL
   in the current Keyman keyboard, because without the dummy prefix key, faking the release of the 
   Alt key will activate the mainmenu of the current application.
 */
-void keybd_sendprefix(LPINPUT pInputs, int *n, BOOL *FPrefix)
+void keybd_sendprefix(LPINPUT pInputs, int *n)
```

Comment:
I have refactored a lot of the code here -- it looks a bit complicated but actually simplifies the use considerably. See `keybd_shift()` for the entry point to all these functions.